content/ember-released-6-3.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Ember 6.3 Released
+authors:
+  - jared-galanis
+date: 2025-03-21T00:00:00.000Z
+tags:
+  - releases
+  - '2025'
+  - version-6-x
+---
+
+Today the Ember project is releasing version 6.3 of Ember.js and Ember CLI.
+
+This release kicks off the 6.4 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 6.3
+
+Ember.js 6.3 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug fixes
+
+Ember.js 6.3 introduced <insert_number> bug fixes.
```

Comment:
```suggestion
Ember.js 6.3 includes 1 noteworthy bug fix:

* [#20825](https://github.com/emberjs/ember.js/pull/20825) Remove the `helper` wrapper from the helper generator; [Plain functions as helpers](https://guides.emberjs.com/release/components/helper-functions/#toc_global-helper-functions) have been supported since v4.5.

```